.ci/jobs/apm-agent-ruby.yml

```diff
@@ -1,4 +0,0 @@
----
```

Comment:
If I remember correctly, I did not remove this because there are other jobs in that folder.

But I guess, it's fine